Y/Yosys/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Yosys"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/YosysHQ/yosys.git", "2e421feb0ea526468493ab008f3c72beb12c2bc6")
+]
+
+dependencies = [
+    Dependency("boost_jll"; compat="=1.76.0"),
+    Dependency("Tcl_jll"),
+    Dependency("Readline_jll"),
+    Dependency("Libffi_jll"; compat="~3.2.2"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd yosys
+if [[ "${target}" == *-apple-* ]] || [[ "${target}" == *-freebsd* ]]; then
+    CONFIG=clang
+else
+    CONFIG=gcc
+fi
+make CONFIG=${CONFIG} PREFIX=${prefix} -j${nproc}
```

Comment:
```suggestion
make CONFIG=${CONFIG} PREFIX=${prefix} LINK_ABC=1 -j${nproc}
```

Reply:
I'd like to try
```suggestion
make CONFIG=${CONFIG} PREFIX=${prefix} ENABLE_ABC=0 -j${nproc}
```
to completely disable ABC, and see what happens, if you're OK